lists/serializers.py

```
@@ -1,15 +1,121 @@
+        fields = [
+            "id",
+            "list_id",
+            "registrant",
+            "registered_by",
+            "status",
+            "submitted_at",
+            "updated_at",
+            "registrant_notes",
+            "admin_notes",
+            "tx_hash",
+        ]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)